README.md

```diff
@@ -10,9 +10,16 @@ Ballerina MySQL Library
 This library provides the functionality required to access and manipulate data stored in a MySQL database.
 
 ### Prerequisite
-Add the MySQL driver JAR as a native library dependency in your Ballerina project's `Ballerina.toml` file.
-It is recommended to use a MySQL driver version greater than 8.0.13 as this library uses the database properties
-from the MySQL driver version 8.0.13 onwards.
+It is required to import the MySQL driver dependency in order to connect to a MySQL database. The `ballerinax/mysql.driver`
+package bundles the latest MySQL driver so that the mysql connector can be used in Ballerina projects easily.
+
+```ballerina
+import ballerinax/mysql.driver as _;
+```
+
+If it is required to use a specific MySQL driver JAR, it can be added as a native library dependency in your Ballerina 
+project's `Ballerina.toml` file. It is recommended to use a MySQL driver version greater than 8.0.13 as this library 
```

Comment:
Shall we add this as a tip?

mysql connector only supports driver versions greater than 8.0.13.

Reply:
Complete rewrite given in https://github.com/ballerina-platform/module-ballerinax-mysql/pull/726/files#r1036139480